prototype_source/inductor_cpp_wrapper_tutorial.rst

```diff
@@ -0,0 +1,83 @@
+Inductor cpp wrapper tutorial
+==============================================================
+
+**Author**: `Chunyuan Wu <https://github.com/chunyuan-w>`_, `Bin Bao <https://github.com/desertfire>`__, `Jiong Gong <https://github.com/jgong5>`__
+
+Prerequisites:
+------------
+-  `torch.compile and TorchInductor concepts in PyTorch <https://pytorch.org/tutorials/intermediate/torch_compile_tutorial.html>`__
+
+Introduction
+------------
+
+Inductor wrapper generates python code to invoke generated kernels and external kernels.
+To reduce the Python overhead of the wrapper and to serve as an intermediate step to support deployment in environments without Python, 
+we implemented cpp wrapper which generates cpp code to combine the generated and external kernels.
+
+
+API
+------------
+This feature is still in prototype stage. To turn it on, the below code change is needed:
+
+.. code:: python
+
+    import torch._inductor.config as config
+    config.cpp_wrapper = True
+
+This will speed up your models by reducing the python overhead of the inductor wrapper.
+
+
+Example code
+------------
+Below is an example of generated code with the default python wrapper:
```

Comment:
Better to add the frontend model code for clarity?

Reply:
The corresponding frontend code has been added.